zipkin-ui/js/component_ui/traceSummary.js

```diff
@@ -227,40 +231,32 @@ export function traceSummariesToMustache(serviceName = null, traceSummaries, utc
     return [];
   } else {
     const traceSummariesCleaned = removeEmptyFromArray(traceSummaries);
-    const maxDuration = Math.max(...traceSummariesCleaned.map((s) => s.duration)) / 1000;
```

Comment:
this was the bug. all the data is in microseconds, yet somehow this line thought it should be millis :P